hudi-client/src/test/java/org/apache/hudi/client/TestTableSchemaEvolution.java

```diff
@@ -408,6 +416,16 @@ public void testCopyOnWriteTable() throws Exception {
     checkReadRecords("000", 2 * numRecords);
   }
 
+  @Test
+  public void testCopyOnWriteTableUsingFileListRollBack() throws Exception {
+    testCopyOnWriteTable(false);
+  }
+
+  @Test
+  public void testCopyOnWriteTableUsingMarkersRollBack() throws Exception {
```

Comment:
these combinations added on Schema evolution test is a bit hard to understand. like why would be testing modes for rollbacks in a schema evolution test? any particular reason?